course05/homework01/martin/string.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <iostream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
+
+
+        bool Empty() const;
+        std::size_t Size() const;
+        int SetString(std::string str);
```

Comment:
In general Setters return void, not int - also not sure what is this int, if it's a size, it should be size_t